### PR TITLE
Analytics doc updates

### PR DESCRIPTION
**Pull request by addb**

### CODE REVIEW

File: using-mobile-extensions/adobe-analytics/README.md [hunk 1]
```diff
@@ -88,19 +88,31 @@ Number of seconds to wait before Analytics launch hits are sent from the SDK. En
...
 
 ```java
 import com.adobe.marketing.mobile.*;
 ```
 {% endtab %}
 
-{% tab title="iOS" %}
-​ 1. Add the library to your project via your Cocoapods `Podfile` by adding `pod 'ACPAnalytics'` ​ 2. Import the Analytics and Identity libraries:
+{% tab title="iOS" %}​
+1. Add the [Mobile Core](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core) and Analytics extensions to your project using Cocoapods. Add following pods in your `Podfile`
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-analytics/README.md]: - the 2nd line sounds like a step...please move to a new line and add a "2."
- Please add a colon (:) after `Podfile`:

File: using-mobile-extensions/adobe-analytics/README.md [hunk 2]
```diff
@@ -88,19 +88,31 @@ Number of seconds to wait before Analytics launch hits are sent from the SDK. En
...
-​ 1. Add the library to your project via your Cocoapods `Podfile` by adding `pod 'ACPAnalytics'` ​ 2. Import the Analytics and Identity libraries:
+{% tab title="iOS" %}​
+1. Add the [Mobile Core](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core) and Analytics extensions to your project using Cocoapods. Add following pods in your `Podfile`  
+
+```objectivec
+pod 'ACPCore'
+pod 'ACPAnalytics'
+``` ​
+
+2. Import the Analytics and Identity libraries:
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-analytics/README.md]: Please change this to "3."

File: using-mobile-extensions/adobe-analytics/README.md [hunk 3]
```diff
@@ -311,7 +323,7 @@ NSMutableDictionary *contextData = [NSMutableDictionary dictionary];
 
 ## Configuration keys
 
-To update the SDK configuration programmatically, use the following information to change your Analytics configuration values. For more information, see [Configuration Methods Reference](./).
+To update the SDK configuration programmatically, use the following information to change your Analytics configuration values. For more information, see [Configuration API Reference](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/configuration/configuration-api-reference).
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-analytics/README.md]: Please structure as follows:
"To update the SDK configuration programmatically, use the following information to change your Analytics configuration values:

(code sample)

For more information, see [Configuration API Reference](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/configuration/configuration-api-reference).

File: using-mobile-extensions/adobe-analytics/analytics-api-reference.md [hunk 4]
```diff
@@ -348,7 +348,7 @@ ACPAnalytics.setVisitorIdentifier("yourVisitorId");
 Before using this API, see [Identify unique visitors](https://docs.adobe.com/content/help/en/analytics/implementation/javascript-implementation/unique-visitors/visid-overview.html).
 {% endhint %}
 
-Sets a custom Analytics visitor identifier. For more information, see [Custom Visitor ID](https://marketing.adobe.com/resources/help/en_US/sc/implement/visid_custom.html).
+Gets a custom Analytics visitor identifier which has been set previously using [setVisitorIdentifier](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics/analytics-api-reference#setidentifier).
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-analytics/analytics-api-reference.md]: - Please add "ThIs API" at the beginning of the 1st sentence...(part of the fixing the gaps project).
- Please add a comma between "identifier" and "which

File: using-mobile-extensions/adobe-analytics/analytics-event-reference.md [hunk 5]
```diff
@@ -117,32 +117,46 @@ The payload definition is composed of the following:
 
 Here is an example of event data for a Track request:
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-analytics/analytics-event-reference.md]: Please lower case the "T" in "Track"

File: using-mobile-extensions/adobe-analytics/README.md (hunk 3 above)
Comment (addb) [reply using-mobile-extensions/adobe-analytics/README.md]: Where does that table go? Should it be like following?

"To update the SDK configuration programmatically, use the following information to change your Analytics configuration values:

that key table 

(code sample)

For more information, see Configuration API Reference.

Comment (rekrishn) [reply using-mobile-extensions/adobe-analytics/README.md]: > Where does that table go? Should it be like following?
> 
> "To update the SDK configuration programmatically, use the following information to change your Analytics configuration values:
> 
> that key table
> 
> (code sample)
> 
> For more information, see Configuration API Reference.

yep @addb...that's correct.

File: using-mobile-extensions/adobe-analytics/analytics-api-reference.md [hunk 6]
```diff
@@ -130,7 +130,7 @@ ACPAnalytics.clearQueue();
 {% endtab %}
 {% endtabs %}
 
-## Get the queue size     <a id="sendqueuedhits"></a>
+## Get the queue size     <a id="getqueuesize"></a>
 
 Retrieves the total number of Analytics hits In the tracking queue.
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-analytics/analytics-api-reference.md]: please lower case "In" to "in"

File: using-mobile-extensions/adobe-analytics/analytics-event-reference.md [hunk 7]
```diff
@@ -353,21 +442,25 @@ Here is the definition of the key-value pairs in this event:
 
 **Forwarding the Analytics server response**
 
-```text
-{ "analyticsserverresponse": "example response to be forwarded" }
+```Javascript
+{
+  "analyticsserverresponse": "example response to be forwarded"
```
Comment (nporter-adbe) [inline using-mobile-extensions/adobe-analytics/analytics-event-reference.md]: Can you add two new fields here? `hitHost` and `hitUrl`?

`"hitHost": "the sever for the analytics hit"`

`"hitUrl": "the url for the analytics hit"`

I'd do it, but I don't want to cause merge conflicts with your PR :)

Comment (rekrishn) [reply using-mobile-extensions/adobe-analytics/analytics-event-reference.md]: no probs @nporter-adbe, done!

**Pull request merged**